addons/web/static/src/legacy/js/widgets/model_field_selector_popover.js

```diff
@@ -429,7 +430,10 @@ var ModelFieldSelectorPopover = Widget.extend({
         var page = this.pages.at(-1);
         this.$(".o_field_selector_popover_header .o_field_selector_title").text(this._getTitle());
 
-        var lines = _.filter(page, this.options.filter);
+
+        //!TODO NEED HELP
+        // this.options.filter is an object like {[arg] : true} 
+        var lines = filter(page, this.options.filter);
```

Comment:
do not create a new utils for this case as it seems to be the only one.
write a simple for loop to filter the lines.